src/lib/utils.js

```diff
@@ -412,13 +412,11 @@ export function fileExists(path: string): boolean {
   }
 }
 
-export function sortByName(packages: Array<any>): string[] {
+export function sortByName(packages: Array<any>, orderAscending: boolean | void = true): string[] {
   return packages.sort(function(a, b) {
-    if (a.name < b.name) {
-      return -1;
-    } else {
-      return 1;
-    }
+    const comparatorNames = a.name.toLowerCase() < b.name.toLowerCase();
```

Comment:
Can we do it like this ? 
*Sort* : mutates the package list here. 
```
return [...packages].sort(({name:a} = {}, {name:b} = {}) => {
	if (a && b) {
		return orderAscending ? (comparatorNames ? -1 : 1) : comparatorNames ? 1 : -1;
	}
  return orderAscending;
})
```

Reply:
In such case, I'd use `slice`, since spread drives me to Flow nightmare with types.

```
return packages.slice().sort(function(a, b) {
    const comparatorNames = a.name.toLowerCase() < b.name.toLowerCase();

    return orderAscending ? (comparatorNames ? -1 : 1) : comparatorNames ? 1 : -1;
  });
```

Reply:
Pushed